test/test_files/tioga-test/tiogatest-nalu.yaml

```diff
@@ -0,0 +1,119 @@
+Simulations:
+  - name: sim1
+    time_integrator: ti_1
+    optimizer: opt1
+
+linear_solvers:
+
+  - name: solve_scalar
+    type: tpetra
```

Comment:
Can we switch these to hypre? These tpetra solver settings are super old and we've not been keeping up with them. I would rather not see another copy of `milestone.xml` put under version control if we can avoid it.